jbmc/src/java_bytecode/bytecode_info.cpp

```diff
@@ -110,116 +163,118 @@ struct bytecode_infot const bytecode_info[]=
+{ nullptr,          0xdd, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xde, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xdf, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xe0, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xe1, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xe2, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xe3, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xe4, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xe5, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xe6, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xe7, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xe8, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xe9, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xea, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xeb, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xec, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xed, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xee, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xef, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xf0, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xf1, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xf2, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xf3, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xf4, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xf5, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xf6, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xf7, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xf8, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xf9, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xfa, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xfb, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xfc, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
+{ nullptr,          0xfd, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
 { "impdep1",        0xfe, ' ', 0, 0, ' ' }, // ; reserved for implementation-dependent operations within debuggers; should not appear in any class file  NOLINT(*)
 { "impdep2",        0xff, ' ', 0, 0, ' ' }, // ; reserved for implementation-dependent operations within debuggers; should not appear in any class file  NOLINT(*)
-{ "wide",           0xc4, ' ', 0, 0, ' ' }, // modifier for others  NOLINT(*)
 { nullptr,          0x00, '\0',0, 0, '\0'}, // zero-initialized NOLINT (*)
```

Comment:
This looks like a magic terminator, but nothing about it is unique (`0x00` is also `nop`, and a `nullptr` mnemonic is shared by the missing entries)-- suggest removing it

Reply:
done